build.rs

```diff
@@ -14,6 +14,7 @@ fn main() {
 
 	let bindings = bindgen::Builder::default()
 		.clang_arg(format!("-I{}/include", dst.display()))
+		.clang_arg("-ISOEM/soem")
```

Comment:
Hello,

Could you please use

```
.clang_arg(format!("-I{}/include/soem", dst.display()))
```

instead. It seems to be more appropriate way. I've checked that it works.

Reply:
Of course :)